maintenance/monitor/monitor-component-metrics.md

```diff
@@ -300,16 +350,17 @@ By executing below commands, you will delete all the resource and services creat
 ```bash
 kubectl delete service felix-metrics-svc -n kube-system
 kubectl delete service typha-metrics-svc -n kube-system
+kubectl delete service kube-controllers-metrics-svc -n kube-system
 kubectl delete namespace calico-monitoring
 kubectl delete ClusterRole calico-prometheus-user
 kubectl delete clusterrolebinding calico-prometheus-user
 ```
 
 ### Best practices
 
-If you enable {{site.prodname}} metrics to Prometheus, a best practice is to use network policy to limit access to the {{site.prodname}} metrics endpoints. For details, see [Secure {{site.prodname}} Prometheus endpoints]({{ site.baseurl }}/security/comms/secure-metrics).  
+If you enable {{site.prodname}} metrics to Prometheus, a best practice is to use network policy to limit access to the {{site.prodname}} metrics endpoints. For details, see [Secure {{site.prodname}} Prometheus endpoints]({{ site.baseurl }}/security/comms/secure-metrics).
```

Comment:
https://docs.projectcalico.org/security/comms/secure-metrics only talks about how to secure calico-node and typha.  I think you'll need to add this new component to it.